ormconfig.ts

```diff
@@ -0,0 +1,22 @@
+import { env } from 'process';
+import { SnakeNamingStrategy } from 'typeorm-naming-strategies';
+
+export default [
+  {
+    name: 'migration',
+    type: env.DB_TYPE,
+    host: env.DB_HOST,
+    port: env.DB_PORT,
+    database: env.DB_NAME,
+    username: env.DB_USER,
+    password: env.DB_PASSWORD,
+    logging: env.ORM_LOGGING === 'true',
+    migrations: [`./src/orm/migrations/**/*{.ts,.js}`],
+    entities: [`./src/orm/entities/**/*.entity{.ts,.js}`],
+    cli: {
+      migrationsDir: `./src/orm/migrations`,
+      entitiesDir: `./src/orm/entities/**/*.entity{.ts, .js}`,
+    },
+    namingStrategy: new SnakeNamingStrategy(),
```

Comment:
let's add:
 ```
     migrationsRun: false,
      logging: ['error'], // it will logs only errors or change it on the env variables level
      synchronize: false,
```

Reply:
we have logging above it will work depending on env.ORM_LOGGING, in some cases i want to see all sql queries